<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Splitter1">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="Splitter2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Panel1" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="Panel1_1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="globalSongLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>71</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>13</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Global songs:</string>
             </property>
             <property name="buddy">
              <cstring>globalSongList</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="globalSongList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="localSongLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>67</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>13</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Local songs:</string>
             </property>
             <property name="buddy">
              <cstring>localSongList</cstring>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>globalSongList</zorder>
          <zorder>globalSongLabel</zorder>
          <zorder>localSongLabel</zorder>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="localSongList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>13</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="selectionRectVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QWidget" name="Panel1_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>71</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>71</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="Running" native="true">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>71</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>71</height>
              </size>
             </property>
             <widget class="QPushButton" name="runButton">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>24</y>
                <width>71</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
             <widget class="QLabel" name="waitText">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>50</y>
                <width>230</width>
                <height>18</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Panel2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QWidget" name="Panel2_1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>189</width>
            <height>322</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>189</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QPushButton" name="removeSongButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>227</y>
             <width>133</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Remove selected song</string>
           </property>
          </widget>
          <widget class="QPushButton" name="playSongButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>292</y>
             <width>133</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Play this song</string>
           </property>
          </widget>
          <widget class="QPushButton" name="upButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>192</y>
             <width>33</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>⇑</string>
           </property>
          </widget>
          <widget class="QPushButton" name="clearSongButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>197</y>
             <width>133</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Clear selected song</string>
           </property>
          </widget>
          <widget class="QPushButton" name="downButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>228</y>
             <width>33</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>⇓</string>
           </property>
          </widget>
          <widget class="QPushButton" name="addSongButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>256</y>
             <width>133</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Add new song</string>
           </property>
          </widget>
          <widget class="QPushButton" name="changeLocalSongButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>168</y>
             <width>133</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Change selected song</string>
           </property>
          </widget>
          <widget class="QPushButton" name="changeGlobalSongButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>45</x>
             <y>22</y>
             <width>132</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Change selected song</string>
           </property>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignRight|Qt::AlignBottom">
         <widget class="QWidget" name="Panel2_2" native="true">
          <property name="minimumSize">
           <size>
            <width>189</width>
            <height>106</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>189</width>
            <height>106</height>
           </size>
          </property>
          <widget class="QCheckBox" name="autoplayCheckbox">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>75</y>
             <width>151</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Autoplay SPC</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="reloadButton">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>0</y>
             <width>139</width>
             <height>23</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Reload list.txt from disk</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="verboseCheckBox">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>29</y>
             <width>171</width>
             <height>21</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Verbose AMK</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="porterModeCheckBox">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>52</y>
             <width>171</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Porter mode</string>
           </property>
          </widget>
          <zorder>reloadButton</zorder>
          <zorder>verboseCheckBox</zorder>
          <zorder>porterModeCheckBox</zorder>
          <zorder>autoplayCheckbox</zorder>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="ResultsPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="resultsText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>globalSongList</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>changeGlobalSongButton</tabstop>
  <tabstop>changeLocalSongButton</tabstop>
  <tabstop>clearSongButton</tabstop>
  <tabstop>addSongButton</tabstop>
  <tabstop>playSongButton</tabstop>
  <tabstop>reloadButton</tabstop>
  <tabstop>porterModeCheckBox</tabstop>
  <tabstop>verboseCheckBox</tabstop>
  <tabstop>autoplayCheckbox</tabstop>
  <tabstop>removeSongButton</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>porterModeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoplayCheckbox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>porterModeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>resultsText</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>init()</signal>
  <slot>runAMK()</slot>
  <slot>changeGlobalSong()</slot>
  <slot>changeLocalSong()</slot>
  <slot>clearLocalSong()</slot>
  <slot>removeLocalSong()</slot>
  <slot>addLocalSong()</slot>
  <slot>playSong()</slot>
  <slot>reloadList()</slot>
  <slot>setVerbose(bool)</slot>
  <slot>setPorterMode(bool)</slot>
  <slot>setAutoplay(bool)</slot>
  <slot>moveSongUp()</slot>
  <slot>moveSongDown()</slot>
 </slots>
</ui>
